app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,61 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, only: %i(edit update)
```

Comment:
three line: 2, 3 and 4 should be combined into one line, because they have the same point: `only: %i(edit update)`